### PR TITLE
Update of the Tracker Online DQM client for Run3 conditions

### PR DESCRIPTION
**Pull request by arossi83**
#### PR description:

Update of the Tracker Online DQM client in order to test Run3 configuration (i.e. transition from SCAL to TCDS)

#### PR validation:

Locally tested with MWGR#1 cosmic run (335507). The flag live need to be changed to "false" on the client under test:

cmsRun "client" dataset=/Cosmics/Commissioning2020-v1/RAW runNumber=335507

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

this is not a backport, a back port to 11_1_X will be done in order to use this change on MWGR#4

### CODE REVIEW

**Comment by arossi83** [comment]
@mmusich @sroychow FYI

**Comment by mmusich** [comment]
@cmsbuild, please test

File: DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 import FWCore.ParameterSet.Config as cms
 
 import sys
-from Configuration.Eras.Era_Run2_2018_pp_on_AA_cff import Run2_2018_pp_on_AA
-process = cms.Process("PIXELDQMLIVE", Run2_2018_pp_on_AA)
+from Configuration.Eras.Era_Run3 import Run3
```
Comment (mmusich) [inline DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py:5]: @arossi83 `_cff` is missing here from `Era_Run3_cff`

**Comment by silviodonato** [comment]
urgent

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by silviodonato** [comment]
merge

**Comment by jfernan2** [comment]
+1

**Pull request merged**